features/samples.feature

```diff
@@ -32,6 +32,8 @@ Feature: Basic smell detection
       DuplicateMethodCall: Inline::C#initialize calls stack.empty? 2 times [https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md]
       DuplicateMethodCall: Module#inline calls Inline.const_get(lang) 2 times [https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md]
       DuplicateMethodCall: Module#inline calls options[:testing] 2 times [https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md]
+      InstanceVariableAssumption: Inline::C assumes too much for instance variable @types, @module_name, @so_name [https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md]
+      InstanceVariableAssumption: Module assumes too much for instance variable @options [https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md]
```

Comment:
Now those changes are fine with me - that's exactly why we're having inline.rb and friends.